EESSI-pilot-2020.09.sh

```diff
@@ -31,6 +31,17 @@ export ARCH=$(uname -m)
 export EESSI_PREFIX=${CVMFS_REPO}/${EESSI_PILOT_VERSION}
 export EPREFIX=${EESSI_PREFIX}/compat/${ARCH}
 
+DETECTION_PARAMETERS=''
+GENERIC=0
+EB='echo eb'
+if [[ "$1" == "--generic" || "$EASYBUILD_OPTARCH" == "GENERIC" ]]; then
+    echo_yellow ">> GENERIC build requested, taking appropriate measures!"
+    DETECTION_PARAMETERS="$DESTINATION_PARAMETERS --generic"
```

Comment:
`s/DESTINATION_PARAMETERS/DETECTION_PARAMETERS/`?

Reply:
This is what I get for using VIM on a remote host! :)